boards/espressif/esp32c6_devkitc/doc/index.rst

```diff
@@ -147,7 +147,7 @@ To build the sample application using sysbuild use the command:
 .. zephyr-app-commands::
    :tool: west
    :zephyr-app: samples/hello_world
-   :board: esp32c6_devkitc
+   :board: esp32c6_devkitc/esp32c6/hpcore
```

Comment:
New target names will break existing users. Would recommend adding an entry to migration guide, and you should also add a deprecated board name for `esp32c6_devkit` to boards/deprecated.cmake aliasing the new hpcore target